library/src/morpheus-master/sass/themes/_light.scss

```diff
@@ -0,0 +1,25 @@
+// Light Theme
+@import "themes/base";
+$primary-color:              $sakai-color-blue-500 !default;
+$alt-colour:                 $sakai-color-orange-400 !default;
+$primary-color-highContrast: $sakai-color-blue-600 !default;	// for use of primary on non-white light backgrounds, such as greys
+$primary-color-shadow:       $sakai-color-blue-600 !default;	// To give primary a darker shadow
```

Comment:
Should the $primary-color-highContrast and $primary-color-shadow have different colour values?

Reply:
I couldn't find the `$primary-color-shadow` being referenced anywhere in the project, so I removed it.

Reply:
Thanks.